src/views/Catalog.vue

```diff
@@ -4,30 +4,42 @@
       <ion-toolbar>
         <ion-title>{{ $t("Catalog") }}</ion-title>
       </ion-toolbar>
-    </ion-header>   
+    </ion-header>
     <ion-content>
-      <ion-searchbar/>
-      <main>        
-        <ion-card v-for="items in 10" :key="items">
-          <Image/>
-            <ion-item lines="none">
-              <ion-label>
-                <p>Brand</p>
-                  Parent name
-                <p>$Sale price</p>
-              </ion-label>
-            </ion-item>          
+      <ion-searchbar v-model="queryString" @keypress.enter="queryString = $event.target.value; getProducts()" />
+      <main>
+        <ion-card v-for="product in products.list" :key="product.productId">
+          <Image :src="product.mainImageUrl" />
+          <ion-item lines="none">
+            <ion-label>
+              <p>{{ product.productId }}</p>
+              {{ product.productName }}
+              <p>${{ product.groupPrice }}</p>
```

Comment:
I think currency should not be hardcoded. Check if there is currency related information available. Also, is there any other variable with price information